app/workers/follows/update_points_worker.rb

```diff
@@ -31,7 +31,7 @@ def calculate_implicit_points(tag, user)
       last_100_long_page_view_article_ids = user.page_views.where(time_tracked_in_seconds: 45..)
         .pluck(:article_id).last(100)
       articles = Article.where(id: last_100_reactable_ids + last_100_long_page_view_article_ids)
-      tags = articles.pluck(:cached_tag_list).flat_map { |list| list.split(", ") }
+      tags = articles.pluck(:cached_tag_list).compact.flat_map { |list| list.split(", ") }
       occurrences = tags.count(tag.name)
       bonus = inverse_popularity_bonus(tag)
       Math.log(occurrences + bonus + 1) # +1 is purelt to avoid log(0) => -infinity
```

Comment:
I know that's not the point of this PR, but any chance we can add this fix for the spelling mistake in the comment, it triggers my OCD :wink:

```suggestion
      Math.log(occurrences + bonus + 1) # +1 is purely to avoid log(0) => -infinity
```

Reply:
always have time to fix a spelling error :)